sdcm/cluster.py

```diff
@@ -3475,13 +3475,13 @@ def execute_cmd(cql_session, entity_type):
                 if filter_out_cdc_log_tables and getattr(row, column_names[1]).endswith(cdc.options.CDC_LOGTABLE_SUFFIX):
                     continue
 
-                if table_name in ["system_schema.dropped_columns", "system.truncated"]:
-                    # skipping those cause of some scylla issues on system tables
-                    # https://github.com/scylladb/scylladb/issues/7186
```

Comment:
This issue was closed because of it a part of https://github.com/scylladb/scylladb/issues/7175.
So actually it is still open.

Reply:
I'm not sure I'm follwoing, I know those are still open, I've just move them below.

the issue with them is only when we try reading from those tables

Reply:
Ignore it. I am after New Year celebration

Reply:
It's probably me that didn't drunk enough :)